libkbfs/block_types.go

```diff
@@ -190,6 +225,27 @@ func (fb FileBlock) DeepCopy(codec kbfscodec.Codec) (*FileBlock, error) {
 	return &fileBlockCopy, nil
 }
 
+// UpdateHash updates the hash of this FileBlock
+func (fb *FileBlock) UpdateHash() kbfshash.RawDefaultHash {
+	fb.cacheMtx.Lock()
+	defer fb.cacheMtx.Unlock()
+	_, hash := kbfshash.DoRawDefaultHash(fb.Contents)
+	fb.hash = &hash
+	return hash
+}
+
+// UpdateHashIfNil updates the hash of this FileBlock only if
```

Comment:
I was confused by the name, since it seems the main purpose of this function is to return the hash, not just to update it.  Can this just be called `GetHash`, and then document in the comment that it calculates and caches the hash if it's nil?